### ISSUE TITLE
Expired Locks Disapear

### ISSUE THREAD

**Issue by burrrata** [1Hive/lock-app#43]
Going through the Lock app to create a user guide. Created a vote, locked some tokens, waited for the lock to be done, but then... once the lock was up the lock disapeared from the UI. I thought the time would change to "done" or something, but the entire thing disapeared. It wasn't until I clicked withdraw that I saw that I had 1 lock with 20 LKT tokens that I could withdraw.

This is incredibly unintuitive as a casual user checking their locks will get no notification or even signal that their lock is ready to be withdrawn. In fact, locks that are ready to be withdrawn are hidden from users and disappear from the main panel.

 It would be great if instead when locks are done locking the timer changed to sign that said: "ready to withdraw"

**Comment by fabriziovigevani** [comment]
Ok i think it makes sense to make it stand out on the screen.

Do you think something like this is more noticeable? 

<img width="1219" alt="Screen Shot 2019-09-07 at 12 40 29 PM" src="https://user-images.githubusercontent.com/22663232/64473909-052d3500-d16d-11e9-8865-2182bf541cfe.png">

Not totally convinced design wise but open to suggestions.

I have added  a notice box saying `you can withdraw your tokens once they unlock` for this same reason so the flow of the app it's clear but i guess we could change the upper box to your suggestion so it' clearer.

**Comment by burrrata** [comment]
Yes! Having the unlocked balance highlighted makes it 10X clearer what is going on and draws attention to the locks that are waiting to be withdrawn.
